vignettes/Estimating-Linear-Intervals.Rmd

```diff
@@ -235,7 +237,7 @@ ames_boot_conf_int %>%
   geom_point(aes(y = Sale_Price),
              alpha = 0.25) +
   scale_x_log10(labels = scales::comma_format()) +
-  scale_y_log10(labels = scales::label_number(scale_cut = scales::cut_short_scale())) +
```

Comment:
These were causing an error. I, for the life of me, could not find a way to resolve it. I might be able to ask Thomas about it.